schema/source_schema.json

```diff
@@ -0,0 +1,416 @@
+          "type": "integer",
+          "minimum": -1
+        },
+        "skiplines": {
+          "type": "integer",
+          "minimum": 1
+        },
+        "notes": {
+          "type": "string"
+        },
+        "id": {
+          "description": "doesn't include prefixed_number or postfixed_street",
+          "oneOf": [
+            { "$ref": "#/definitions/base_conform_field_value" },
+            { "$ref": "#/definitions/function_regexp" },
+            { "$ref": "#/definitions/function_remove_prefix" },
+            { "$ref": "#/definitions/function_remove_postfix" },
+            { "$ref": "#/definitions/function_join" },
+            { "$ref": "#/definitions/function_format" }
+          ]
+        },
+        "number": {
+          "description": "doesn't include postfixed_street",
+          "oneOf": [
+            { "$ref": "#/definitions/base_conform_field_value" },
+            { "$ref": "#/definitions/function_regexp" },
+            { "$ref": "#/definitions/function_prefixed_number" },
+            { "$ref": "#/definitions/function_remove_prefix" },
+            { "$ref": "#/definitions/function_remove_postfix" },
+            { "$ref": "#/definitions/function_join" },
+            { "$ref": "#/definitions/function_format" }
+          ]
+        },
+        "street": {
+          "description": "doesn't include prefixed_number",
+          "oneOf": [
+            { "$ref": "#/definitions/base_conform_field_value" },
+            { "$ref": "#/definitions/function_regexp" },
+            { "$ref": "#/definitions/function_postfixed_street" },
+            { "$ref": "#/definitions/function_remove_prefix" },
+            { "$ref": "#/definitions/function_remove_postfix" },
+            { "$ref": "#/definitions/function_join" },
+            { "$ref": "#/definitions/function_format" }
+          ]
+        },
+        "unit": {
+          "description": "doesn't include prefixed_number or postfixed_street",
+          "oneOf": [
+            { "$ref": "#/definitions/base_conform_field_value" },
+            { "$ref": "#/definitions/function_regexp" },
+            { "$ref": "#/definitions/function_remove_prefix" },
+            { "$ref": "#/definitions/function_remove_postfix" },
+            { "$ref": "#/definitions/function_join" },
+            { "$ref": "#/definitions/function_format" }
+          ]
+        },
+        "city": {
+          "description": "doesn't include prefixed_number or postfixed_street",
+          "oneOf": [
+            { "$ref": "#/definitions/base_conform_field_value" },
+            { "$ref": "#/definitions/function_regexp" },
+            { "$ref": "#/definitions/function_remove_prefix" },
+            { "$ref": "#/definitions/function_remove_postfix" },
+            { "$ref": "#/definitions/function_join" },
+            { "$ref": "#/definitions/function_format" }
+          ]
+        },
+        "district": {
+          "description": "doesn't include prefixed_number or postfixed_street",
```

Comment:
These `description`s should be more useful information about the field, not descriptions about the list in `oneOf`.